Tests/defaultslots.spec.js

```diff
@@ -11,15 +11,15 @@ let airport;
 
 //2nd Test
 console.log('--------------------');
-console.log('2. slotsOccupied array has a default number of strings from Plane that can be changed');
+console.log('2. slotsOccupied array has a default number of 10');
 
 //Arrange
 airport = new Airport();
 expected = 10;
 
 //Act
-airport.slotsAvailable(10)
-actule = airport.slotsOccupied.length = 10;
```

Comment:
Spelt actual wrong and also realised that this was completely the wrong way to this.